castervoice/asynch/hmc_rules/hmc_launch_rule.py

```diff
@@ -9,7 +9,7 @@
 
 def receive_settings(data):
     settings.SETTINGS = data
-    settings.save_config()
+    settings.save_config(paths = True)
```

Comment:
This one keeps the code the same for hmc. hmc is not really used as far as I know so this is just a conservative approach